src/fixml/cli/checklist.py

```diff
@@ -4,40 +4,47 @@
 class ChecklistActions:
 
     @staticmethod
-    def convert(from_path: str, to_path: str, overwrite: bool = False) -> None:
+    def convert(to_path: str, checklist_path: str = None,
```

Comment:
@SoloSynth1 are there any reasons why `self` is not part of the arguments in those methods?

Reply:
The method does not use the class's other methods and properties and hence can be made into static method (hence the decorator `@staticmethod`)